modules/backend/widgets/calendar/assets/js/src/Calendar.js

```
@@ -0,0 +1,297 @@
+            info.jsEvent.preventDefault();
+            const url = info.event.url;
+            if (url) {
+                if (url.startsWith('http') || (!url.startsWith('$'))) {
+                    location.href = url;
+                } else {
+                    const elements = url.split('.');
+                    let funcName = elements.pop(); // remove the last element
+                    const objectName = elements.join('.');
+                    const index = funcName.indexOf('(');
+                    funcName = funcName.substring(0, index);
+                    const object = eval(objectName); // eslint-disable-line no-eval
```

Biome (2.5.6)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)